stages/org.osbuild.grub2

```diff
@@ -84,6 +85,16 @@ def write_grub_cfg(tree, path):
                   "blscfg\n")
 
 
+def efi_vendor_dir(tree):
```

Comment:
I'd make this also a stage option instead (possibly verifying the correctness if we want). Would make sense to me to discover and include this info in image-info though.

In general, we should try to think of the tree as "write-only" when we can (doesn't quite work always, of course).

Reply:
I was thinking inside `options` have either:
```
{
  "modes": ["uefi", "legacy"],
  "uefi": {"vendor": "redhat"},
}
```
or just the presence of the `uefi` and/or `legacy` key triggers the mode:
```
{
  "uefi": {"vendor": "fedora"},
  "legacy": true # could later become a dict with options...
}
```

Reply:
@larskarlitski and I prefer the second so if nobody screams STOP I will do the second version...